docs/releases/v2021.1.rst

```diff
@@ -0,0 +1,141 @@
+Gluon 2021.1
+============
+
+Important notes
+---------------
+
+Upgrades to v2021.1 and later releases are only supported from releases v2018.2 and later. This is due migrations that have been removed to simplify maintenance.
+
+
+Added hardware support
+----------------------
+
+
+ath79-generic
+~~~~~~~~~~~~~
+
+* Plasma Cloud
+
+    - PA300 [#outdoor]_
+    - PA300E [#outdoor]_
+
+* TP-Link
+
+    - Archer C2 v3
+    - Archer D50 v1
+
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+* AVM
+
+    - FRITZ!Box 7530
+
+* Plasma Cloud
+
+    - PA1200 [#outdoor]_
+    - PA2200
+
+
+ramips-mt7620
+~~~~~~~~~~~~~
+
+* Netgear
+
+    - EX3700
+    - EX3800
+
+
+.. [#outdoor]
+  This device is supposed to be set up outdoors and will therefore have its outdoor mode flag automatically enabled.
+
+
+Major changes
+-------------
+
+Multicast optimizations (batman-adv)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this release we reenable the multicast optimizations, that have gone through another round of bug squashing upstream. This allows, together with the ``noflood`` patchset, that multicast traffic will be converted to unicast traffic, when the number of subscribers does not exceed 16. This is beneficial as we would previously drop multicast traffic that wasn't explicitly allowed, but now can allow multicast traffic if it wouldn't result in broadcast.
```

Comment:
```suggestion
In this release, we reenable the multicast optimizations, that have gone through another round of bug squashing upstream. This allows, together with the ``noflood`` patchset, that multicast traffic will be converted to unicast traffic when the number of subscribers does not exceed 16. This is beneficial as we would previously drop multicast traffic that wasn't explicitly allowed but now can allow multicast traffic if it wouldn't result in broadcast.
```

Reply:
Next to @kaechele's suggestion, I'd propose to add a note regarding the impact on layer 2 overhead (which was the primary goal of this feature for the Gluon use-case):

> In this release, we reenable the multicast optimizations, that have gone through another round of bug squashing upstream. For one thing this reduces layer 2 overhead, especially for IPv6 Neighbor Discovery ("ARP" for IPv6). For another this allows, together with the ``noflood`` patchset, [...]